index.bs

```diff
@@ -78,15 +79,20 @@ This specification uses terminology from the Fetch, HTML, HTTP, and URL standard
 
 <h2 id="semantics">Passkey Endpoints URLs</h2>
 
-To advertise support for [=passkeys=] and/or provide direct endpoints for passkey creation and management, [=Relying Party|Relying Parties=] MUST host a JSON document at the path formed by concatenating the string `.well-known/passkey-endpoints` with the `https` [=scheme=] and [=relying party identifier=] as per [[WELL-KNOWN]]. A redirect MUST not be returned.
+To advertise support for [=passkeys=] and/or provide direct endpoints for passkey creation and management, [=Relying Party|Relying Parties=] MUST return a response at the path formed by concatenating the string `.well-known/passkey-endpoints` with the `https` [=scheme=] and [=relying party identifier=] as per [[WELL-KNOWN]]. 
 
-<p class=example>The passkey endpoints URL for [=RP ID=] `"example.com"` is `"https://example.com/.well-known/passkey-endpoints"`.
+<p class=example id="endpoint-url-example">The passkey endpoints URL for [=RP ID=] `"example.com"` is `"https://example.com/.well-known/passkey-endpoints"`.
 
 <h3 id="response">Server Response</h3>
 
 The server in this context is a [=WebAuthn Relying Party=] (RP) supporting [=passkeys=].
 
-A successful response MUST use the 200 OK HTTP status code and return a JSON object using the `application/json` content type.
+A successful response MUST return either:
+- the 200 OK HTTP status code and a JSON object using the `application/json` content type
+- a [=redirect status=] of 302, 303, or 307, and a [=Location=] header.
```

Comment:
Is there a reason why `HTTP 301 Moved Permanently` is omitted from this list? Is it bad practice in the spec sense? Otherwise the list looks fine to me.